tests/Geppetto/README.md

```diff
@@ -0,0 +1,55 @@
+[![Prestashop](https://i.imgur.com/qDqeQ1E.png)](https://www.prestashop.com)
+
+![Geppetto](./media/logo.png)
+
+# How to Install Geppetto project
+Clone this repo and run them directy with a simple `node` command.
+
+```bash
+git clone https://github.com/mbadrani/geppettov2.git
+cd geppettov2
```

Comment:
Doc to be updated with new sub folders

Reply:
done in my next PR, thanks

Reply:
You don't need to create a new pull request, you can commit directly in the branch.